internal/jobexecutor/job_executor_test.go

```diff
@@ -654,7 +718,10 @@ func TestJobExecutor_Execute(t *testing.T) {
 		executor.Execute(ctx)
 		riversharedtest.WaitOrTimeout(t, bundle.updateCh)
 
-		job, err := bundle.exec.JobGetByID(ctx, bundle.jobRow.ID)
+		job, err := bundle.exec.JobGetByID(ctx, &riverdriver.JobGetByIDParams{
+			ID:     bundle.jobRow.ID,
+			Schema: "",
+		})
```

Comment:
These all get quite a bit more verbose, unfortunately 😕

Reply:
Yeah :/ I think we can clean a lot of this up again with a few more helpers for test functions that end up generating a lot of objects.